daos/mysql/Items.php

```diff
@@ -329,11 +329,11 @@ public function hasThumbnail($thumbnail) {
      * @param string $icon file
      */
     public function hasIcon($icon) {
-        $res = \F3::get('db')->exec('SELECT count(*) AS amount
-                   FROM '.\F3::get('db_prefix').'items 
-                   WHERE icon=:icon',
+        $res = \F3::get('db')->exec('SELECT id
+                   FROM items
+                   WHERE icon=:icon limit 1',
                   array(':icon' => $icon));
-        return $res[0]['amount']>0;
+        return sizeof($res)>0;
```

Comment:
As a PHP programmer, I haven’t heard of sizeof until now and I would expect it to do something like sizeof in C. It is, however, just an alias of count, could you replace it?